src/database/CMakeLists.txt

```diff
@@ -1,5 +1,29 @@
-add_library(example-lib Example.cpp)
+cmake_minimum_required(VERSION 3.12)
 
-target_include_directories(example-lib PRIVATE
-    ${INCLUDE_DIR}
+add_library(database-lib database.cpp)
+add_executable(database-exec database.cpp)
```

Comment:
What is a purpose of the executable?

Reply:
only for test purposes. Just to be sure that everything works

Reply:
If so, maybe adding a test for it could be better.